add z-index to correctly position map in demo

talimarcus opened the pull request:
Fixes https://github.com/predix-ui/predix-ui.github.io/issues/178

davidrleonard commented:
This is good to go. Merge and add a tag when you're ready. Thanks.

davidrleonard commented:
Actually hold up. Shouldn't the style variable default to auto? Otherwise this might be a breaking API change if it introduces a change in how the map renders for customers. 

Also seems like we should minor, not patch, since we are adding functionality. 

Thoughts?

talimarcus commented:
Fixed that @davidrleonard and removed the updates to HISTORY.md

Leaving here for now:

* Adds the `--px-map-z-index` property to `px-map` which can be used to position
  the map in the correct layer within your application. Note that the z-indices
  of Leaflet layers exist in their own stacking context, and so are not affected
  by the z-index set on px-map. Read more about the stacking context [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context).
* Adds the `invalidateSize()` method to `px-map`, which can be used to check
  whether the map container has changed size or visibility, and - if so - updates
  the map accordingly by refreshing the tile layer.
